README.md

```diff
@@ -49,8 +63,11 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
   </form>
 ```
 
+  var user_input = $('#submit').val();
```

Comment:
Actually you need to get what was entered in the `color` box, not the `submit` box. So `var user_input = $('#color').val()`.